src/earn/prepareTransactions.test.ts

```diff
@@ -132,91 +130,139 @@ describe('prepareTransactions', () => {
   describe('prepareWithdrawAndClaimTransactions', () => {
     beforeEach(() => {
       jest.mocked(encodeFunctionData).mockReturnValue('0xencodedData')
+      jest.mocked(triggerShortcutRequest).mockResolvedValueOnce({
+        transactions: [
+          {
+            from: '0x123',
+            to: '0x567',
+            data: '0xencodedData',
+            gas: '50200',
+            estimatedGasUse: '49900',
+          },
+        ],
+      })
+      jest.mocked(triggerShortcutRequest).mockResolvedValueOnce({
+        transactions: [
+          {
+            from: '0x1234',
+            to: '0x5678',
+            data: '0xencodedData',
+            gas: '50100',
+            estimatedGasUse: '49800',
+          },
+        ],
+      })
     })
 
     it('prepares withdraw and claim transactions with gas subsidy on', async () => {
-      const rewardsTokens = mockRewardsPositions[1].tokens
       jest.mocked(isGasSubsidizedForNetwork).mockReturnValue(true)
 
       const result = await prepareWithdrawAndClaimTransactions({
-        amount: '5',
-        token: mockToken,
         walletAddress: '0x1234',
         feeCurrencies: [mockFeeCurrency],
-        rewardsTokens,
-        poolTokenAddress: '0x5678',
+        pool: mockEarnPositions[0],
+        hooksApiUrl: 'https://hooks.api',
+        positionsWithClaimableRewards: [mockRewardsPositions[1] as unknown as ClaimablePosition],
```

Comment:
why `as unknown`?

Reply:
I guess this is because it not really is a ClaimablePosition and we're just forcing TS to identify this as a ClaimablePosition. Would be nice if we can avoid this.